src/test_application/OSPMagnum.cpp

```diff
@@ -160,102 +171,97 @@ void testapp::config_controls(OSPMagnum& rOspApp)
     }
 }
 
-
 //Map for all the keys
 //The tuple is in this order: device, number, and hold/pressed
-using Key_t = OSPMagnum::KeyEvent::Key;
-using Mouse_t = OSPMagnum::MouseEvent::Button;
-using VarOp_t = osp::ButtonVarConfig::VarOperator;
-using VarTrig_t = osp::ButtonVarConfig::VarTrigger;
 
 typedef std::tuple<int, int> button_tuple;
```

Comment:
consider using std::pair instead of std::tuple here. Less template machinery for the compiler to deal with.